content/articles/api-authentication-with-django-knox-and-postman-testing/index.md

```diff
@@ -0,0 +1,343 @@
+![hero](/engineering-education/api-authentication-with-django-knox-and-postman-testing/hero.png)
+
+#### Introduction
+
+Django-Knox is a framework that makes the authentication of the API endpoints built with the Django Rest Framework easier. Although Knox is also a token-based authentication like JSON Web Token (J.W.T) auth. This comes with well-detailed documentation for easy implementation.
```

Comment:
Change "Although Knox is also a token-based authentication like JSON Web Token (J.W.T) auth. This comes with well-detailed documentation for easy implementation." to "However, Knox is also a token-based authentication like JSON Web Token (JWT) auth. Django-Knox comes with well-detailed documentation for easy implementation."